pkg/justinstall/registry.go

```diff
@@ -283,7 +285,25 @@ func (e *RegistryEntry) install(path string) error {
 	case "zip":
 		log.Println("Extracting to", e.destination())
 
-		return installer.ExtractZIP(path, e.destination())
+		if err := installer.ExtractZIP(path, e.destination()); err != nil {
+			return err
+		}
+
+		if shortcuts, prs := e.Installer.options()["shortcuts"]; prs {
+			for _, shortcut := range shortcuts.([]interface{}) {
+				shortcutName := expandString(shortcut.(map[string]interface{})["name"].(string), nil)
+				shortcutTarget := expandString(os.ExpandEnv(shortcut.(map[string]interface{})["target"].(string)), nil)
+				shortcutLocation := filepath.Join(startMenu, shortcutName + ".lnk")
+
+				log.Println("Creating shortcut to %s in %s", shortcutTarget, shortcutLocation)
```

Comment:
```suggestion
				log.Println("Creating shortcut to", shortcutTarget, "in", shortcutLocation)
```

Reply:
Done

Reply:
Also, for the future, note that you meant to use `Printf` with a `\n`, as `Println` doesn't support format arguments.